source/_variables/replacements.py

```diff
@@ -163,6 +163,9 @@
     "|SPLUNK_LATEST_MINOR|" : "8.2",
     "|WAZUH_SPLUNK_REV_CURRENT_LATEST|" : "1", # 8.2
     "|WAZUH_SPLUNK_REV_CURRENT_8.1|" : "1",
+    #
+    # === Node.js
+    "|NODE_VERSION|": "18.19.0",

```

Comment:
**suggestion**: if the variable is only related to Wazuh dashboard or we think it could have a conflict with other elements of Wazuh, then we could add the app reference to the variable name.